### DISCUSSION TITLE
Is the amount of RAM disk that can be used simultaneously limited?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3299] [Q&A Feedback] [answered]
Is there a technical limitation on how many RAM disks can be used by Sandboxie simultaneously?

Other limitations, as the amount of RAM, that is. I am just curious if users could create and run like 50 of them at a time.

**Comment by offhub** [comment, marked as answer]
> By using volume mount points, you can graft or mount a target partition onto a folder on another physical disk. You can also exceed the 26-letter limitation for drive letter references. [*](https://learn.microsoft.com/en-us/troubleshoot/windows-server/high-availability/configure-volume-mount-points-server-cluser)

I don't know if there are any limitations set by ImDisk.

**Reply by bastik-1001** [reply]
This sounds good, in either case. If David does not say that it is limited, due to how it is handled, you answered my question. So I will mark this as answered later, when David said something about this.
